src/datachain/data_storage/schema.py

```diff
@@ -150,14 +151,12 @@ class DataTable:
     def __init__(
         self,
         name: str,
-        engine: "Engine",
-        metadata: Optional["sa.MetaData"] = None,
```

Comment:
I'm not sure what was the case for it to be `None` and for this code below:

`self.metadata: sa.MetaData = metadata if metadata is not None else sa.MetaData()` 

that initializes its own copy of `MetaData()`

I haven't found any use case so far. It is simpler this way when internals of the `DatabaseEngine` are not exposed into DataTable.

Let's see if tests pass.